src/main/java/org/jvnet/hudson/plugins/thinbackup/backup/HudsonBackup.java

```diff
@@ -222,6 +223,17 @@ public void backup() throws IOException {
             moveOldBackupsToZipFile(backupDirectory);
             removeSuperfluousBackupSets();
         }
+        touchCompleteFile();
+    }
+
+    /**
+     * Creates a empty file backup-completed.info at the end of the backup.
+     *
+     * @throws IOException if an I/O Error occurs
+     */
+    public void touchCompleteFile() throws IOException {
+        File backupCompletedFile = new File(backupDirectory.getAbsolutePath(), COMPLETED_BACKUP_FILE);
+        backupCompletedFile.createNewFile();
```

Comment:
```suggestion
        FileUtils.touch(backupCompletedFile);
```
This will also work if there is, for whatever reason also a file like this present.